db/update-files.d/1_alter_raster_metadata.sql

```diff
@@ -0,0 +1,24 @@
+\c sauber_data
+
+CREATE TABLE image_mosaics.raster_metadata (
```

Comment:
Just for clarification: The `raster_metadata` table is no more created in the base SQL script?

Reply:
Good point. 
Was originally: DROP TABLE raster_metadata, then recreate. 
Now: CREATE **IF NOT EXISTS** to make sure it does before adding a column.
Changed in newest commit.